src/main.js

```diff
@@ -0,0 +1,24 @@
+import core from "@actions/core";
+import { start, stop } from "./lib/ec2Handler.js";
+
+async function run() {
+  try {
+    const instances_id = JSON.parse(core.getInput('instances_id', {required: true}))
+    const action = (core.getInput('action', {required: true})).toLowerCase
+
+    if (action === "start") {
+      await start(instances_id);
+      return
+    }
+
+    if (action === "stop") {
+      await stop(instances_id);
+      return
+    }
```

Comment:
```suggestion
    } else { throw new Error('Invalid action. It must be either `start` or `stop`)
    }
```

Reply:
I'm doing an early return if the action match on one of the any we have, as default it will return error at the end using the github toolkit `core.setFailed`, I use this instead of throwing an error with `throw` because it will handle the `throw` and will add the label with the error on the actions summary.

```js
    if (action.toLowerCase() === "start") {
      await start(instances_id);
      return
    }

    if (action.toLowerCase() === "stop") {
      await stop(instances_id);
      return
    }

core.setFailed("Invalid action. Please provide either 'start' or 'stop' as action")
```